src/Component/Nav.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.css';
+
+const Nav = () => (
+  <nav className="nav">
+    <div className="header">Math Magician</div>
+    <ul className="link">
+      <li>
+        <Link to="/Home" className="font">Home</Link>
```

Comment:
- [ ] Kindly modify the link on the Navbar to be the root path.

  ```
  <Link to="/" className="font">Home</Link>
  ```

Reply:
Thanks for the review